templates/snmpd/snmpd.conf.erb

```diff
@@ -5,23 +5,23 @@
 ################################################################################
 # AGENT BEHAVIOUR
 
-<% if @master != nil -%>
+<% if @master.nil? -%>
 <% if @master -%>
```

Comment:
We have a convention of indentations that make the ERB code easier to read:
<% if condition -%>
<%   if nested_condition -%>
<%   end -%>
<% end -%>